build/stage-release-appcenter.yml

```diff
@@ -42,7 +43,7 @@ jobs:
   environment: ${{ parameters.deploymentEnvironment }}
 
   variables:
-  - group: ApplicationTemplate.Distribution.AppStore
+  - group: ${{ parameters.iOSVariableGroup }}
```

Comment:
I'm pretty sure that's not even needed for AppCenter iOS

Reply:
updated